homeassistant/components/buienradar/sensor.py

```diff
@@ -774,13 +774,7 @@ def _load_data(self, data):  # noqa: C901
         self._measured = data.get(MEASURED)
         sensor_type = self.entity_description.key
 
-        if (
```

Comment:
This one comes up a lot and reviews, super nice to not have to do it manually now